kolibri/core/assets/src/core_app.js

```diff
@@ -43,4 +43,4 @@ var Kolibri = function() {
     _.extend(this, _.pick(mediator, public_methods));
 };
 
-module.exports = Kolibri;
+module.exports = new Kolibri();
```

Comment:
Except that we are setting any references to `require('./core_app')` to the `kolibriGlobal` object, so require doesn't actually get executed as such.